app/controllers/api/generic_works_controller.rb

```diff
@@ -0,0 +1,92 @@
+module API
+  # Adds an endpoint that customizes the update action on nested works
+  class GenericWorksController < ApplicationController
+    before_action :my_load_and_authorize_resource, only: [:update, :destroy, :show, :add_parent, :remove_parent]
```

Comment:
So I guess it was assumed that the functionality from another API controller was to be retained in this one, but that assumption could easily incorrect. 

https://github.com/projecthydra/sufia/blob/master/app/controllers/api/items_controller.rb#L9

we'll make the change.